package.json

```diff
@@ -57,8 +57,8 @@
     "@firebase/database": "^0.6.0",
     "@types/node": "^8.10.59",
     "dicer": "^0.3.0",
-    "jsonwebtoken": "8.1.0",
-    "node-forge": "0.7.4"
+    "jsonwebtoken": "^8.5.1",
+    "node-forge": "^0.7.6"
```

Comment:
Any reason why we wouldn't update this to most recent`0.9.1`?

Reply:
I upgraded via `npm update` command, and this is what was automatically picked by NPM (possibly because 0.x versions are allowed to make breaking changes between minor releases). I will test 0.9.1 and upgrade in a separate PR.

Reply:
Sounds good. Thanks for checking!